docs/development/qubes_staging.rst

```diff
@@ -324,3 +324,66 @@ to provision staging VMs on-demand. To remove the staging instance, use the Mole
 .. code:: sh
 
    molecule destroy -s qubes-staging
+
+Accessing the Journalist Interface (Staging) in Whonix-based VMs
```

Comment:
It's worth noting that if this machine is also used for SecureDrop Workstation testing and development, there is a quicker way to do this:
https://github.com/freedomofpress/securedrop-workstation/wiki/Developer-Tips#how-to-connect-to-the-journalist-interface-in-qubes

Reply:
Good flag, although for the staging VMs I treat the URLs are pretty ephemeral, whereas my Workstation setup usually connects to a long-running hardware instance, with stable URLs. In other words, I think it's worth capturing the additional detail that @rocodes has provided here. Wouldn't be opposed to a link-out as you suggest, of course